great_expectations/rule_based_profiler/types/data_assistant_result/data_assistant_result.py

```diff
@@ -236,12 +236,259 @@ def get_expect_domain_values_to_be_between_chart(
             domain_type=domain_type,
         )
 
+        anomaly_coded_line: alt.Chart = (
+            DataAssistantResult._determine_anomaly_coded_line(
+                line, tooltip, metric_name
+            )
+        )
+
+        return band + lower_limit + upper_limit + anomaly_coded_line
+
+    @staticmethod
+    def get_vertically_concatenated_chart(
+        dfs_by_column: List[Tuple[str, pd.DataFrame]],
+        metric_name: str,
+        metric_type: alt.StandardType,
+        domain_name: str,
+        domain_type: alt.StandardType,
+    ) -> alt.VConcatChart:
+        """
+        Args:
+            dfs_by_column: A list of tuples pairing pandas dataframes with the columns they correspond to
+            metric_name: The name of the metric as it exists in the pandas dataframe
+            metric_type: The altair data type for the metric being plotted
+            domain_name: The name of the domain as it exists in the pandas dataframe
+            domain_type: The altair data type for the domain being plotted
+
+        Returns:
+            A vertically concatenated (vconcat) altair line chart
+        """
+        charts: List[alt.Chart] = []
+
+        i: int
+        column_name: str
+        df: pd.DataFrame
+        for i, (column_name, df) in enumerate(dfs_by_column):
+            include_title: bool = i == 0
+            chart: alt.Chart = DataAssistantResult._get_vertically_concatenated_chart(
+                df=df,
+                column_name=column_name,
+                metric_name=metric_name,
+                metric_type=metric_type,
+                domain_name=domain_name,
+                domain_type=domain_type,
+                include_title=include_title,
+            )
+            charts.append(chart)
+
+        return alt.vconcat(*charts)
+
+    @staticmethod
+    def _get_vertically_concatenated_chart(
```

Comment:
Maybe rename to `_get_vertically_concatenated_line_chart` is more precise?

Reply:
done!